Source/Database/DBAPIManager.swift

```diff
@@ -123,7 +124,7 @@ class DBAPIManager: NSObject {
         do {
             return try dbHandler?.getAllHighlights(byBookId: id, page: page) ?? []
         } catch {
-            print(dbHandler?.errorMessage ?? "")
+            os_log("getAllHighlight error: %@", dbHandler?.errorMessage ?? "")
```

Comment:
I wonder if these log messages in the various catch blocks should be flagged as .error instead of whatever the default is.

Reply:
👍  will add .error